osf_tests/test_node.py

```diff
@@ -1709,12 +1709,17 @@ def test_find_for_user():
     noncontrib = UserFactory()
     Contributor.objects.create(node=node1, user=contrib)
     Contributor.objects.create(node=node2, user=contrib)
-    assert node1 in Node.find_for_user(contrib)
-    assert node2 in Node.find_for_user(contrib)
-    assert node1 not in Node.find_for_user(noncontrib)
 
-    assert node1 in Node.find_for_user(contrib, Q(is_public=False))
-    assert node2 not in Node.find_for_user(contrib, Q(is_public=False))
+    contrib_nodes = Node.find_for_user(contrib)
```

Comment:
Eventually we should get rid of `find_for_user`. It doesn't provide anything that isn't already possible with `.nodes`.

Reply:
I'm pretty sure I can get rid of this now. I think I didn't originally to avoid overlap with other PRs.